### PR TITLE
PTV-1165 - add Prediction Class to the download 

### PR DESCRIPTION
**Pull request by landml**
@cshenry, @JamesJeffryes, @samseaver  - Add Prediction Class to the download. Also rename columns to match data browser. Just making it easier for users to find the data they are looking for.

### CODE REVIEW

File: lib/Bio/KBase/ObjectAPI/KBasePhenotypes/PhenotypeSimulationSet.pm [hunk 1]
```diff
@@ -73,10 +73,11 @@ sub export {
 sub printTSV {
     my $self = shift;
 	my $args = Bio::KBase::ObjectAPI::utilities::args([], {file => 0,path => undef}, @_);
-	my $output = ["geneko\tmediaws\tmedia\taddtlCpd\taddtlCpdBounds\tcustomReactionBounds\tgrowth\tsimulated growth\tsimulated growth fraction\tgapfilled reaction count\tgapfilled reactions"];
+	my  $output = ["Growth condition\tGenekKO\tmediaws\tAddtional Compounds\taddtlCpdBounds\tcustomReactionBounds\tObserved Normal growth\tSimulated growth\tsimulated growth fraction\tgapfilled reaction count\tgapfilled reactions\tPrediction class"];
```
Comment (JamesJeffryes) [inline lib/Bio/KBase/ObjectAPI/KBasePhenotypes/PhenotypeSimulationSet.pm]: We should make this consistently CamelCase or Title Case

Comment (JamesJeffryes) [reply lib/Bio/KBase/ObjectAPI/KBasePhenotypes/PhenotypeSimulationSet.pm]: Pretty sure there's no reimport of the data so I'd lean toward the latter

Comment (landml) [reply lib/Bio/KBase/ObjectAPI/KBasePhenotypes/PhenotypeSimulationSet.pm]: My objective was to match the data viewer for all fields that they have in common.

Comment (landml) [reply lib/Bio/KBase/ObjectAPI/KBasePhenotypes/PhenotypeSimulationSet.pm]: I can change them all to Title case if you want.

**Comment by JamesJeffryes** [comment]
Hmm, I think I'd rather it be internally consistent than match the viewer
(we should fix the viewer though)

**Comment by landml** [comment]
I'm working on PTV-1165 which only asks for the prediction class in the download. I moved and renamed the 'media' column because up to this point, the user was looking as 'Growth conditions' as the first column. When things get renamed and moved around it is harder to line them up with what you saw before.

Just let me know what you think I should do to move this forward.

**Comment by landml** [comment]
@JamesJeffryes - I've changed the column heading to be a consistent Title case. The public ticket that this goes with is PUBLIC-571.

**Review by JamesJeffryes — APPROVED** [review]
Looks OK to me. Got the 👍 from Chris too.

**Comment by landml** [comment]
Super. Just need to get it released.

**Pull request merged**